addons/iot_drivers/tools/system.py

```diff
@@ -212,37 +213,29 @@ def _get_system_uptime():
         return float(f.readline().split()[0])
 
 
-def is_ngrok_enabled():
-    """Check if a ngrok tunnel is active on the IoT Box"""
-    try:
-        response = requests.get("http://localhost:4040/api/tunnels", timeout=5)
-        response.raise_for_status()
-        response.json()
-        return True
-    except (requests.exceptions.RequestException, ValueError):
-        # if the request fails or the response is not valid JSON,
-        # it means ngrok is not enabled or not running
-        _logger.debug("Ngrok isn't running.", exc_info=True)
-        return False
+def is_remote_debug_enabled():
+    """Check if the tailscale connection is up on the IoT Box"""
+    p = subprocess.run(['tailscale', 'status'], check=False)
+    return p.returncode == 0
 
 
-def toggle_remote_connection(token=""):
-    """Enable/disable remote connection to the IoT Box using ngrok.
-    If the token is provided, it will set up ngrok with the
-    given authtoken, else it will disable the ngrok service.
+def toggle_remote_debug(auth_key=""):
+    """Enable/disable remote connection to the IoT Box using tailscale.
 
-    :param str token: The ngrok authtoken to use for the connection"""
-    _logger.info("Toggling remote connection with token: %s...", token[:5] if token else "<No Token>")
-    p = subprocess.run(
-        ['sudo', 'ngrok', 'config', 'add-authtoken', token, '--config', '/home/pi/ngrok.yml'],
-        check=False,
-    )
-    if p.returncode == 0:
+    :param auth_key: The auth key to use to enable remote access.
+    If empty, remote access will be disabled.
+    """
+    _logger.info("%s remote access", 'enabling' if auth_key else 'disabling')
+
+    server_url = re.sub(r'^https?://|/|:', '', get_conf('remote_server') or 'iotbox')
```

Comment:
Let's add iot box identifier for databases with multiple iot boxes
```suggestion
    from urllib.parse import urlparse
    from helpers import get_identifier
    
    id = get_identifier()
    server_url = urlparse(get_conf('remote_server')).netloc + f'-{id}', or f'iotbox-{id}')
```